src/solver/conv_hip_implicit_gemm_grouped_fwd_xdlops.cpp

```diff
@@ -285,7 +284,8 @@ bool PerformanceConfigHipImplicitGemmGroupFwdXdlops::RunParameterPredictionModel
     }
     return false;
 }
-#endif
+#endif // MIOPEN_ENABLE_AI_KERNEL_TUNING
+#endif // MIOPEN_BACKEND_HIP && MIOPEN_USE_COMPOSABLEKERNEL
```

Comment:
Is it really so that AI thing depends on CK???

Reply:
This particular solver? Yes.

Reply:
Oops, I see now. Obvious.